File: Lec -1 C Programming/points.c
#include <stdio.h>
#include <cs50.c>
#include <cs50.h>

int main (void)
{
    const int MINE = 34;
    int points = get_int("How much points did you get? ");

    if (points < MINE)
    {
        printf("You get less than me!\n");
    }
    else if (points > MINE)
    {
        printf("You get more than me!\n");
    }
    else 
    {
        printf("You get exactly same as me!\n");
    }
}

File: Lec - 6 Python/points.py
import cs50

points = cs50.get_int("How many points did you lose? ")

if points < 2 :
    print("You lose fewer points than me")
    
elif points > 2 :
    print("You lose more points than me")
    
else :
    print("You lose the same amount of points as me")